h/renderers.py

```diff
@@ -0,0 +1,31 @@
+# Taken from https://pyramid-cookbook.readthedocs.org/en/latest/templates/customrenderers.html
```

Comment:
Could you just add a `# -*- coding: utf-8 -*-` comment to the top of this new file?